core-api/src/controllers/access.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from "express";
+import { AccessService } from "../services/access.service";
+import { HttpStatus } from "../utils/httpStatus";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import Messages from "../utils/messages";
+
+export class AccessController {
+    public static async createAccess(req: Request, res: Response) {
+        const { place_id, permission_id } = req.body;
+        if (!place_id || ! permission_id) {
+            res.status(HttpStatus.BAD_REQUEST).json(new ServerError(
+                Messages.validation.data_needs_to_be_provided,
+                ErrorCode.NOT_ENOUGH_DATA,
+            ).toJSON());
+
+            return;
+        } else if (!isNan(place_id) || !isNan(permission_id)){
```

Comment:
The function name is `isNaN`, this is probably throwing a ReferenceError.
Also, it should be like this:
```js
if (isNaN(place_id) || isNaN(permission_id) {
    res.status(HttpStatus.BAD_REQUEST).json(new ServerError(
        Messages.validation.ids_should_be_numbers,
        ErrorCode.NOT_ENOUGH_DATA,
    ).toJSON());
}

```

Reply:
fixed